test/honest_test.go

```diff
@@ -52,13 +53,14 @@ func TestHonest_ChainAgreement(t *testing.T) {
 			tsg := sim.NewTipSetGenerator(tipSetGeneratorSeed)
 			baseChain := generateECChain(t, tsg)
 			targetChain := baseChain.Extend(tsg.Sample())
-			test.options = append(test.options, sim.WithBaseChain(&baseChain), sim.WithTipSetGenerator(tsg))
+			test.options = append(test.options,
+				sim.WithBaseChain(&baseChain),
+				sim.AddHonestParticipant(test.participantCount, sim.NewFixedECChainGenerator(targetChain)))
```

Comment:
Is it a limitation of the chain generator that it can't be re-used between participants? I couldn't tell from the interface/docs. Did you inline it here and below out of necessity, or just style?

Reply:
It is [reused](https://github.com/filecoin-project/go-f3/blob/2953820a50c9df397205e902912063d7b1db33fe/sim/host.go#L43) across participants: a generator instance is associated to a number of participants, i.e. they call generate chain on the same instance. 

What am I missing?

Reply:
I didn't put this comment in a good place, but was wondering why in some places (e.g. test/absent_test.go) you moved the generator from a variable to inline. But sounds like it's nothing.